rhel7/fixes/bash/disable_prelink.sh

```diff
@@ -1,16 +1,12 @@
 # platform = Red Hat Enterprise Linux 7
-#
-# Disable prelinking altogether
-#
-if grep -q ^PRELINKING /etc/sysconfig/prelink
-then
-  sed -i 's/PRELINKING.*/PRELINKING=no/g' /etc/sysconfig/prelink
-else
-  echo -e "\n# Set PRELINKING=no per security requirements" >> /etc/sysconfig/prelink
-  echo "PRELINKING=no" >> /etc/sysconfig/prelink
-fi
 
-#
-# Undo previous prelink changes to binaries
-#
-/usr/sbin/prelink -ua
+if rpm --quiet -q prelink; then
+	if grep -q ^PRELINKING /etc/sysconfig/prelink
+	then
+		sed -i 's/PRELINKING.*/PRELINKING=no/g' /etc/sysconfig/prelink
+	else
+		echo -e "\n# Set PRELINKING=no per security requirements" >> /etc/sysconfig/prelink
```

Comment:
Shellcheck will complain about `\n` inside double quotes, so it is better to change those to single quotes.
https://github.com/koalaman/shellcheck/wiki/SC1117